src/main.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './styles/droneList.css'
+
+const drones = [
```

Comment:
I would create a models directory, where i would create:
```
type Drone = {
    id: number;
    status: DroneStatus;
};
```
And the DroneStatus
`type DroneStatus = "parked" | "delivering" etc...;`

Reply:
then i would assign the Drone type to the drones constant

Reply:
Done